### PR TITLE
Prepare for the 0.15.0 release

### PR DESCRIPTION
**Pull request by arajkumar**
## Description

Fixes https://github.com/timescale/promscale/issues/1670

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ var (
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
 
-	Promscale                  = "0.15.0-dev.0"
+	Promscale                  = "0.15.0"
```
[the remark this answers is not in the document]
Comment (arajkumar) [reply pkg/version/version.go:41]: I think it is better to mandate 0.7.0 for this release as the extension release contains critical bug fixes like https://github.com/timescale/promscale_extension/pull/526. WDYT?

File: migration-tool/cmd/prom-migrator/main.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ const (
 	defaultStartTime       = "1970-01-01T00:00:00+00:00" // RFC3339 based time.Unix from 0 seconds.
 	defaultMaxReadDuration = time.Hour * 2
 	defaultLaIncrement     = time.Minute
-	version                = "0.0.5"
+	version                = "0.0.6"
```
Comment (niksajakovljevic) [inline migration-tool/cmd/prom-migrator/main.go:38]: Why are we bumping prom-migrator (i believe it should have not dependency on Promscale)?

File: pkg/version/version.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (niksajakovljevic) [reply pkg/version/version.go:41]: I believe that without 0.7.0 people will be seeing error message related to recently added vacuum engine so I think it's valid to mandate 0.7.0

File: migration-tool/cmd/prom-migrator/main.go (hunk 2 above)
Comment (arajkumar) [reply migration-tool/cmd/prom-migrator/main.go:38]: @alejandrodnm did few changes in prom-migrator[1], hence I bumped it's version

[1] https://github.com/timescale/promscale/commits/master/migration-tool

**Review by niksajakovljevic — APPROVED** [review]
LGTM. Since helm chart was recently moved to a separate repo I wonder if we should do anything on that front?

**Review by cevian — APPROVED** [review]

**Comment by arajkumar** [comment]
> LGTM. Since helm chart was recently moved to a separate repo I wonder if we should do anything on that front?

As @paulfantom mentioned during the call, it is all automated for the helm charts .

**Pull request merged**